lib/is_external_link.js

```diff
@@ -18,17 +10,19 @@ const urlObj = (str) => {
 
 function isExternalLink(url) {
   const { config } = this;
-  const exclude = Array.isArray(config.external_link.exclude) ? config.external_link.exclude
-    : [config.external_link.exclude];
-  const data = urlObj(url);
-  const host = data.hostname;
-  const sitehost = typeof urlObj(config.url) === 'object' ? urlObj(config.url).hostname : config.url;
+  const sitehost = parse(config.url).hostname || config.url;
+  // Pass a base to new URL
```

Comment:
I would say the purpose is `// Workaround for relative url`.

Reply:
Updated to `// handle relative url` since there is a `handle mailto: javascript: vbscript: and so on` below.